ambry-api/src/main/java/com.github.ambry/rest/RestUtils.java

```diff
@@ -458,4 +481,71 @@ public static Long getTimeFromDateString(String dateString) {
   public static long toSecondsPrecisionInMs(long ms) {
     return ms - (ms % 1000);
   }
+
+  /**
+   * Gets the value of the header {@code header} in {@code args}.
+   * @param args a map of arguments to be used to look for {@code header}.
+   * @param header the name of the header.
+   * @param required if {@code true}, {@link IllegalArgumentException} will be thrown if {@code header} is not present
+   *                 in {@code args}.
+   * @return the value of {@code header} in {@code args} if it exists. If it does not exist and {@code required} is
+   *          {@code false}, then returns null.
+   * @throws RestServiceException if {@code required} is {@code true} and {@code header} does not exist in
+   *                                    {@code args} or if there is more than one value for {@code header} in
+   *                                    {@code args}.
+   */
+  private static String getHeader(Map<String, Object> args, String header, boolean required)
```

Comment:
No changes to the function, I just moved the private methods to the bottom of the class.